grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -126,22 +128,125 @@ public GrpcServiceBuilder addService(ServerServiceDefinition service) {
         return this;
     }
 
+    /**
+     * Adds a gRPC {@link ServerServiceDefinition} to this {@link GrpcServiceBuilder}, such as
+     * what's returned by {@link BindableService#bindService()}.
+     *
+     * <p>Note that the specified {@code path} replaces the normal gRPC service path.
+     * Let's say you have the following gRPC service definition.
+     * <pre>{@code
+     * package example.grpc.hello;
+     *
+     * service HelloService {
+     *   rpc Hello (HelloRequest) returns (HelloReply) {}
+     * }}</pre>
+     * The normal gRPC service path for the {@code Hello} method is {@code "/example.grpc.hello/Hello"}.
+     * However if you set the {@code path} to {@code "/foo"}, the {@code Hello} method will be served at
+     * {@code "/foo/Hello"}. This is useful for supporting unframed gRPC with HTTP idiomatic path.
+     */
+    public GrpcServiceBuilder addService(String path, ServerServiceDefinition service) {
+        registryBuilder.addService(requireNonNull(path, "path"), requireNonNull(service, "service"));
+        return this;
+    }
+
+    /**
+     * Adds a {@linkplain MethodDescriptor method} of gRPC {@link ServerServiceDefinition} to this
+     * {@link GrpcServiceBuilder}. You can get {@link MethodDescriptor}s from the enclosing class of
+     * your generated stub.
+     *
+     * <p>Note that the specified {@code path} replaces the normal gRPC service path.
+     * Let's say you have the following gRPC service definition.
+     * <pre>{@code
+     * package example.grpc.hello;
+     *
+     * service HelloService {
+     *   rpc Hello (HelloRequest) returns (HelloReply) {}
+     * }}</pre>
+     * The normal gRPC service path for the {@code Hello} method is {@code "/example.grpc.hello/Hello"}.
```

Comment:
Isn't this `{@code "/example.grpc.hello.HelloService/Hello"}`?

Reply:
Oops...